cli/src/account.rs

```diff
@@ -24,11 +28,50 @@ use crate::{
     println_log_error,
 };
 
+#[derive(Helper, Completer, Hinter, Validator)]
+pub struct PromptHelper {
+    #[rustyline(Completer)]
+    completer: AccountCompleter,
+    #[rustyline(Hinter)]
+    hinter: HistoryHinter,
+    colored_prompt: String,
+}
+
+impl Highlighter for PromptHelper {
+    fn highlight_prompt<'b, 's: 'b, 'p: 'b>(&'s self, prompt: &'p str, default: bool) -> Cow<'b, str> {
+        if default {
+            Cow::Borrowed(&self.colored_prompt)
+        } else {
+            Cow::Borrowed(prompt)
+        }
+    }
+
+    fn highlight_hint<'h>(&self, hint: &'h str) -> Cow<'h, str> {
+        Cow::Owned(hint.bold().to_string())
+    }
+}
+
 // loop on the account prompt
 pub async fn account_prompt(wallet: &Wallet, mut account: Account) -> Result<(), Error> {
-    let mut history = AccountHistory::default();
+    let config = Config::builder()
+        .auto_add_history(true)
+        .history_ignore_space(true)
+        .completion_type(rustyline::CompletionType::List)
+        .edit_mode(rustyline::EditMode::Emacs)
+        .build();
+
+    // Panic: should never happen
+    let mut rl = Editor::with_history(config, MemHistory::with_config(config)).unwrap();
+
+    let helper = PromptHelper {
+        completer: AccountCompleter,
+        hinter: HistoryHinter {},
```

Comment:
No new fn?

Reply:
or default?

Reply:
Actually couldn't you impl `Default` on PromptHelper too?

Reply:
probably :+1:

Reply:
Errrrr what the hell rustyline, why would you define a struct like this 😆 
```rust
pub struct HistoryHinter {}
```